### ISSUE TITLE
Contact formulation

### ISSUE THREAD

**Issue by bhajay** [MatMechLab/AsFem#21]
It would help to add support for contact formulations.

**Comment by yangbai90** [comment]
> It would help to add support for contact formulations.

Hi Dr. Harish, thank you very much for the interest in AsFem. For 'contact formulation', you mean the **Contact Mechanics** problem, right?
Currently, I have no experiences with the **Contact** problem, what I plan to do for the first step is adding the cohesive zone element for interface delamination. Afterward, I will try to consider the contact issues at the interface. It would be very helpful if you could point me to any articles or lectures that I could learn about the contact problem.

BR
Yang

**Comment by bhajay** [comment]
Yup. I was meaning contact mechanics. I primarily work in contact & impact mechanics and hence was interested. Sorry for the delayed replies
